src/dbConnect.ts

```diff
@@ -0,0 +1,19 @@
+import dotenv from 'dotenv';
+import mongoose from 'mongoose';
+
+dotenv.config()
+const dbConnectionString = process.env.MONGODB_URI as string;
+
+const dbConnect = async() => {
+    try {
+        await mongoose.connect(dbConnectionString);
+        console.log('Connected to MongoDB')
+    } catch (error) {
+        let message
+        if (error instanceof Error) message = error.message
+        else message = String(error);
+        console.log('Could not connect to Database: ' + message)
+    }
+};
+
+export default dbConnect;
```

Comment:
imo it's best not to use default exports, it makes refactoring harder because inevitably someone will change the name of the thing when they import it.